File: LeetCode/1436.py
#Link: https://leetcode.com/problems/destination-city/

class Solution:
    def destCity(self, paths: List[List[str]]) -> str:
        for i in range(len(paths)):
            for j in range(len(paths)):
                if paths[i][0] == paths[j][1]:
                    paths[j][1] = "0"
        for i in range(len(paths)):
            if paths[i][1] != "0":
                return paths[i][1]
                

File: LeetCode/cpp/1436.cpp
//Link: https://leetcode.com/problems/destination-city/

class Solution {
public:
    string destCity(vector<vector<string>>& paths) {
        for(int i = 0 ; i < paths.size() ; i++)
            for(int j = 0 ; j < paths.size() ; j++)
                if(paths[i][0] == paths[j][1])
                    paths[j][1] = "0";
        for(int i = 0 ; i < paths.size() ; i++)
            if(paths[i][1] != "0")
                return paths[i][1];
        return "";
    }
};
